test/e2e/flowaggregator_test.go

```diff
@@ -852,7 +852,7 @@ func getUint64FieldFromRecord(t *testing.T, record string, field string) uint64
 func getCollectorOutput(t *testing.T, srcIP, dstIP, srcPort string, isDstService bool, checkAllRecords bool, isIPv6 bool) (string, []string) {
 	var collectorOutput string
 	var recordSlices []string
-	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout, func() (bool, error) {
+	err := wait.PollImmediate(500*time.Millisecond, aggregatorInactiveFlowRecordTimeout+exporterActiveFlowExportTimeout, func() (bool, error) {
```

Comment:
would be good to have comments in the code to explain the timeout choice when it's not obvious

Reply:
Thanks, added.